modelci/types/bo/dynamic_profile_result_bo.py

```diff
@@ -13,57 +13,67 @@
 
 class ProfileMemory(object):
     """Memory class in dynamic profiling result business object.
+
+    Args:
+        total_memory (int): Total memory in Byte.
+        memory_usage (int): GPU memory used in Byte.
+        utilization (float): GPU utilization.
     """
 
-    def __init__(self, memory: int, cpu_memory: int, gpu_memory: int):
-        """Initializer.
-
-        Args:
-            memory (int): Total memory in Byte.
-            cpu_memory (int): CPU memory in Byte.
-            gpu_memory (int): GPU memory in Byte.
-        """
-        # total memory
-        self.memory = memory
-        # CPU memory
-        self.cpu_memory = cpu_memory
-        # GPU memory
-        self.gpu_memory = gpu_memory
+    def __init__(self, total_memory: int, memory_usage: int, utilization: float):
+        """Initializer."""
+        # total GPU memory
+        self.total_memory = total_memory
```

Comment:
better use self.gpu_total_memory since we may profile TPU in the future

Reply:
TPU profile result is recorded separately with GPU profile result, shall keep using total memory:
for profiling result on a GPU
```
total memory: xxx Byte
...
```
for profiling result on a TPU
```
total memory: xxx Byte
...
```

Reply:
Agree with @YuanmingLeee , currently our profiler calls the docker container with only single device attached, the memory represents in the same way.

Reply:
Ok.